pom.xml

```diff
@@ -52,6 +52,11 @@
             <name>Public JBoss release repository</name>
             <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2</url>
         </repository>
+        <snapshotRepository>
+            <id>jboss-qa-snapshots</id>
+            <name>JBoss QA snapshot repository</name>
+            <url>http://${env.NEXUS_HOST}/nexus/content/repositories/jboss-qa-snapshots/</url>
+        </snapshotRepository>
```

Comment:
@sgala this should be removed before the merge

Reply:
Agreed. This is only needed to deploy snapshots, and even then it can be substituted by arguments to the mvn call.